packages/web/stories/feedback/tooltip/Tooltip.stories.ts

```diff
@@ -14,31 +14,56 @@ const Template: Story<FreudTooltipComponent> = (
   `,
 });
 
-export const Top = Template.bind({});
-Top.args = {
-  tooltipLabel: 'Tooltip label top',
-  tooltipPosition: 'top'
+//Teste
+export const TooltipTest = Template.bind({});
+
+// Position
+export const Top = () => {
+  return {
+    template: `
+      <span freud-tooltip tooltipLabel="Tooltip label top" tooltipPosition="top" style="font-family: 'Source Sans Pro'">Passe o mouse aqui!</span>
```

Comment:
Não deveria precisar de nenhum `style` para o componente `freud-tooltip` em si.

Se for para deixar o texto de dentro com a font-family `Source Sans Pro', sugiro ou criar um `<span style="font-family: 'Source Sans Pro', sans-serif;">Passe o mouse aqui!</span>` dentro com essa estilização, ou refatorar o exemplo para usar o `freud-typography`.

O mais simples é fazer o primeiro:

```suggestion
      <span freud-tooltip tooltipLabel="Tooltip label top" tooltipPosition="top">
  <span style="font-family: 'Source Sans Pro', sans-serif;">Passe o mouse aqui!</span>
</span>
```

Reply:
Vale também para as outras stories desse componente.

Reply:
Não mudei isso, pq faz parte da task https://zenklub.atlassian.net/browse/TPLA-314
A princípio deixei assim para agilizar o processo e liberar a tarefa de migração, mas esses pontos que você sugeriu vão ser modificados posteriormente, deixei comentado na descrição da TPLA-314, se quiser deixar mais sugestões lá também, pq não sei se serei eu quem vai pegar essa task.